core/src/main/scala/sttp/tapir/Schema.scala

```diff
@@ -123,8 +125,11 @@ object Schema extends SchemaExtensions with SchemaMagnoliaDerivation with LowPri
   implicit def schemaForMap[V: Schema]: Schema[Map[String, V]] = macro SchemaMapMacro.schemaForMap[Map[String, V], V]
 
   def oneOfUsingField[E, V](extractor: E => V, asString: V => String)(mapping: (V, Schema[_])*): Schema[E] = macro oneOfMacro[E, V]
+
+  def derive[T]: Schema[T] = macro Magnolia.gen[T]
```

Comment:
Why is `Schema.derive` defined here, but for validator it's in `ValidatorMagnoliaDerivation`?

Reply:
It is now in `Validator`

Reply:
thank you!